.github/workflows/build-artifacts.yml

```
@@ -0,0 +1,64 @@
+name: Build GFM artifacts
+
+# Builds the GFM library as artifacts that can be used by other workflows
+#
+# It checks for any existing artifacts with an expiry date at least 10 days after today
+# If none are found, the library is built and uploaded.
+
+# This workflow can be run on demand, but is intended to be run on a weekly schedule
+# This should ensure that there is always at least one artifact available but no more than 2
+
+on:
+  # Schedule once a week
+  schedule:
+    - cron: '11 12 * * 0'
+  push:
+    paths:
+      - '**/build-artifacts.yml' # self
+      - 'pelican/build-cmark.sh'
+  workflow_dispatch:
+
+jobs:
+  build-artifact:
+    runs-on: ubuntu-latest
+    strategy:
+      # Allow for multiple versions to be maintained
+      matrix:
+        gfm_version: ['0.28.3.gfm.12']
```

```suggestion
        gfm_version:
        - '0.28.3.gfm.12'
```

Please use some form of the array notation so that when people add / remove items they aren't blamed for changing everything.